CraftTweaker2-API/src/main/java/crafttweaker/runtime/CrTTweaker.java

```diff
@@ -73,6 +75,9 @@ public boolean loadScript(boolean isSyntaxCommand, String loaderName) {
         
         scriptFiles.sort(PreprocessorManager.SCRIPT_FILE_COMPARATOR);
         
+        Map<String, byte[]> classes = new HashMap<>();
+        IEnvironmentGlobal environmentGlobal = GlobalRegistry.makeGlobalEnvironment(classes);
+        
```

Comment:
Moved Global Envo creation outside the loop so the Global envo will be global for all scripts (loaded with that specific loader!)